specificatie/gba-genereervariant/openapi.yaml

```diff
@@ -321,7 +321,9 @@ components:
         type:
           type: string
         fields:
-          $ref: '#/components/schemas/Fields'
+          type: array
+          items:
```

Comment:
hierbij verwacht ik ook een minItems van 1 en wellicht ook een maxItems?

Reply:
Willen we dan ook een maximum stellen aan het aantal fields dat gevraag wordt ? Zo ja, wat is dan een zinnig aantal maxItems ? 
Eens met de minItems.

Reply:
de API heeft nu minder dan 130 velden in de API, wanneer je inOnderzoek, groepen, geheim en opschorting niet meetelt. Dus iets in die orde grootte zou ik denken.

Reply:
Besloten is om inderdaad een maxItems van 130 te definiëren.